requirements/dev.txt

```diff
@@ -416,14 +416,11 @@ fluent-runtime==0.3 \
     --hash=sha256:8d41d4bf4cb19039370e14de78f2991cae9721d44e66165989a69ff5c22d90f0 \
     --hash=sha256:d8a81a36d111ba3e66af27f9224e94ef305cf3bec407c3f98a7c597eef554652
     # via -r requirements/prod.txt
-fluent-syntax==0.17.0 \
-    --hash=sha256:ac3db2f77d62b032fdf1f17ef5c390b7801a9e9fb58d41eca3825c0d47b88d79 \
-    --hash=sha256:e26be470aeebe4badd84f7bb0b648414e0f2ef95d26e5336d634af99e402ea61
-    # via -r requirements/prod.txt
 fluent-syntax==0.17.0 \
     --hash=sha256:ac3db2f77d62b032fdf1f17ef5c390b7801a9e9fb58d41eca3825c0d47b88d79 \
     --hash=sha256:e26be470aeebe4badd84f7bb0b648414e0f2ef95d26e5336d634af99e402ea61
     # via
+    #   -r requirements/prod.txt
     #   compare-locales
     #   fluent-runtime
```

Comment:
Never noticed that was duplicated but looks like pip-tools maybe fixed that bug. 👍